docs/content/integrations/dbt-cloud.mdx

```diff
@@ -28,78 +46,74 @@ width={1055}
 
-# configure an operation to run the specific job
-run_dbt_nightly_sync = dbt_cloud_run_op.configured(
-    {"job_id": 33333}, name="run_dbt_nightly_sync"
-)
+```python startafter=start_define_dbt_cloud_instance endbefore=end_define_dbt_cloud_instance file=/integrations/dbt/dbt_cloud.py dedent=4
+from dagster_dbt import dbt_cloud_resource
 
-# configure a resource to connect to your dbt Cloud instance
-my_dbt_cloud_resource = dbt_cloud_resource.configured(
-    {"auth_token": {"env": "DBT_CLOUD_AUTH_TOKEN"}, "account_id": 11111}
+dbt_cloud_instance = dbt_cloud_resource.configured(
+    {
+        "auth_token": {"env": "DBT_CLOUD_API_TOKEN"},
+        "account_id": {"env": "DBT_CLOUD_ACCOUNT_ID"},
+    }
 )
-
-# create a job that uses your op and resource
-@job(resource_defs={"dbt_cloud": my_dbt_cloud_resource})
-def my_dbt_cloud_job():
-    run_dbt_nightly_sync()
 ```
 
----
+## Step 2: Loading dbt Cloud models as assets
 
-## Running a dbt Cloud job after another op completes
+In this step, you'll load the dbt Cloud models managed by a dbt Cloud job into Dagster as assets. For context, a dbt Cloud job defines set of commands to run for a dbt Cloud project.
 
-The `dbt_cloud_run_op` has an optional `start_after` input. If you supply the output of another operation to this input, the dbt Cloud op will not start until that upstream operation successfully completes:
+Using our dbt Cloud resource, we can retrieve information about the models that the dbt Cloud job is managing.
 
-```python startafter=start_dbt_cloud_job2 endbefore=end_dbt_cloud_job2 file=/integrations/dbt/dbt_cloud.py dedent=4
-@job(resource_defs={"dbt_cloud": my_dbt_cloud_resource})
-def my_two_op_job():
-    run_dbt_nightly_sync(start_after=another_op())
+```python startafter=start_load_assets_from_dbt_cloud_job endbefore=end_load_assets_from_dbt_cloud_job file=/integrations/dbt/dbt_cloud.py dedent=4
+from dagster_dbt import load_assets_from_dbt_cloud_job
+
+# Use the dbt_cloud_instance resource we defined in Step 1, and the job_id from Prerequisites
+dbt_cloud_assets = load_assets_from_dbt_cloud_job(
+    dbt_cloud=dbt_cloud_instance,
+    job_id=33333,
+)
 ```
 
----
+The <PyObject module="dagster_dbt" object="load_assets_from_dbt_cloud_job" /> function loads the dbt Cloud models into Dagster as assets, creating one Dagster asset for each model.
+
+When invoked, the function:
+
+1. Invokes your dbt Cloud job with command overrides to compile your dbt project,
+2. Parses the metadata provided by dbt Cloud, and
+3. Generates a set of software-defined assets reflecting the models in the project. These assets share the same underlying op, which will invoke dbt Cloud to run the dbt Cloud job represented by the loaded assets.
```

Comment:
```suggestion
3. Generates a set of software-defined assets reflecting the models in the project managed by the dbt Cloud job. Materializing these assets will invoke dbt Cloud to run the dbt Cloud job represented by the loaded assets.
```

Reply:
My suggestion is mostly a reflection of confusion, I don't understand what "these assets share the same underlying op" implies for the use of these assets. 

I think it'd also be worth adding a little more color about what happens when assets are materialized, e.g. does the dbt  Cloud job run verbatim, or do we only execute `dbt run`? If the dbt Cloud job includes tests, does asset materialization?